Add README and initial resource sharing plan

dhimmel opened the pull request:
This resource sharing plan is based on one @cgreene and I wrote for a grant that @cgreene and @bdsullivan are submitting to the NIH.

The main goal is to ensure that project outputs are openly licensed. 

Who [owns academic data/content is complicated](http://osc.universityofcalifornia.edu/2016/09/who-owns-your-data/), so these resource sharing plans should ensure open licensing regardless of owner.

Another goal is to prevent license incompatibilities down the road. Hence, the creators and investigators are provided the ability to release content as CC0.

Comments please!

dhimmel committed:
Add initial resource sharing plan

dhimmel committed:
Update persistent archive example language

dhimmel commented:
@jakevdp, I saw your [tweet on git for prose](https://twitter.com/jakevdp/status/796817091684769792).

Does the formatting of prose in this PR look good to you? Any suggestions to make maintaining prose easier?

ethanwhite commented on README.md:
```diff
@@ -0,0 +1,18 @@
+The goal of the plan is to ensure project content created under the grant is openly released.
+
+In instances where the University owns content produced under the grant, a well-crafted resource sharing plan is often the best way to ensure the open licensing of content.
+Since Universities are generally the recipient of the grant and approve its content, a resource sharing plan can legally obligate a University allow the researchers to openly license their research.
```
"a University TO allow" or "that a University allow" ?

ethanwhite reviewed (commented):
I think this looks awesome. My one major suggestion is to think about where exactly this will end up for most funders and whether that means it might be worthwhile trying to shorten it. For NSF This would to in the Data Management plan. In my experience those aren't too tight on space so probably not a big deal, but if this needed to go in the proposal body (which it might have to in some cases; I've needed to do it that way in the past) it would be way too long. One option would be to add 2nd, 2-3 sentence version, designed to cover the 2nd use case.

dhimmel committed:
Fix typo: missing 'to'

dhimmel committed:
Add reuse section to the README

jakevdp commented:
> Does the formatting of prose in this PR look good to you?

Looks pretty good!

cgreene commented:
To go along with @ethanwhite's comment, you may want to specify that this is for NIH grants (where it does not count towards page limits). Maybe prefix `resource-sharing-plan.md` with nih so you have `nih-resource-sharing-plan.md`. Then as people work this into other grants, they may be willing to commit back ones that fit other funder requirements.

strasser commented:
I like it! There isn't as much detail as I would like to see in a data sharing plan from grantees, so you might also add that this might not be sufficient for all funders. Perhaps point to DCC or DMPTool as resource. Might ask Stephanie Simms from CDL/DMPtool to look? Can connect if you'd like

dhimmel committed:
Move plans to examples directory

dhimmel committed:
Solicit community contributions

cgreene commented on README.md:
```diff
@@ -1,20 +1,32 @@
+A good resource sharing plan helps avoid unanticipated hurdles and disagreements when it's time to release project content.
+An explicit plan ensures that the Universities, Investigators, and Funders involved in a grant are all on the same page regarding resource sharing.
+Having a straightforward licensing arrangement can avoid delays down the road when seeking licensing approval from your University.
+Resource sharing plans can impose legally-binding obligations for University's to openly license research.
```
University's -> universities

cgreene commented on README.md:
```diff
@@ -1,20 +1,32 @@
+An explicit plan ensures that the Universities, Investigators, and Funders involved in a grant are all on the same page regarding resource sharing.
+Having a straightforward licensing arrangement can avoid delays down the road when seeking licensing approval from your University.
+Resource sharing plans can impose legally-binding obligations for University's to openly license research.
+Finally, a plan that commits to openness can make your proposal more attractive to funders.
```
During our recent labcarpentry discussion, we decided that our resources would be opinionated. I think you can change `can` to `should` here if you want to. This isn't a mandatory change.

cgreene commented on README.md:
```diff
@@ -1,20 +1,32 @@
+You are not required to attribute the resource sharing plans to use them.
 However, feedback is appreciated.
-If you receive comments on your grant related to these resource sharing plans, we'd love to know.
+For example, if you receive comments on your grant related to these resource sharing plans, we'd love to know.
```
How should somebody provide feedback?

cgreene reviewed (changes requested):
A few changes that I noticed.

dhimmel committed:
Add background to NIH-example.md

Mentions DMPTools and DCC.
https://github.com/lab-carpentry/blueprint-resourcesharing/issues/2

dhimmel committed:
Address @cgreene review

https://github.com/lab-carpentry/blueprint-resourcesharing/pull/1#pullrequestreview-12332569

cgreene commented on README.md:
```diff
@@ -10,16 +10,16 @@ Examples are located in [`examples`](examples).
 A good resource sharing plan helps avoid unanticipated hurdles and disagreements when it's time to release project content.
 An explicit plan ensures that the Universities, Investigators, and Funders involved in a grant are all on the same page regarding resource sharing.
 Having a straightforward licensing arrangement can avoid delays down the road when seeking licensing approval from your University.
```
Unless it's a specific university, these should be lowercase.

dhimmel commented:
> you may want to specify that this is for NIH grants (where it does not count towards page limits).

@cgreene, done

> My one major suggestion is to think about where exactly this will end up for most funders and whether that means it might be worthwhile trying to shorten it.

@ethanwhite I changed the repository to be a collection of many examples. This initial example is for the NIH, where the resource sharing plan doesn't count as part of the grant size limit. Hopefully, we can get contributions with plans for alternative funders.

> There isn't as much detail as I would like to see in a data sharing plan from grantees, so you might also add that this might not be sufficient for all funders.

@strasser, I added the following to the prologue of the example:

> This plan largely focuses on ensuring the open licensing of project outputs. Carly Strasser — a Program Officer for the [Data-Driven Discovery Initiative](https://www.moore.org/initiative-strategy-detail?initiativeId=data-driven-discovery) at the Gordon & Betty Moore Foundation — [suggests](https://github.com/lab-carpentry/blueprint-resourcesharing/pull/1#issuecomment-266153553) adding more details regarding your specific data outputs and corresponding sharing plans. For inspiration, see the [Digital Curation Centre guide](http://www.dcc.ac.uk/resources/how-guides/develop-data-plan "How to Develop a Data Management and Sharing Plan") or [public Data Management Plans](https://dmptool.org/public_dmps "DMPTool").

cgreene reviewed (approved):
One minor comment around capitalization. After addressing this, it will LGTM 👍 Probably good to give others some time to give additional input before merging.

dhimmel committed:
Fix capitalization

cgreene commented:
@dhimmel : when you merge, can you either file a PR or an issue against the website repo? We'll want to edit https://github.com/lab-carpentry/lab-carpentry.github.io/blob/master/pages/blueprints.md to link to this resource.
